include/aspect/utilities.h

```diff
@@ -1069,6 +1057,45 @@ namespace aspect
     template <int dim>
     SymmetricTensor<2,dim> nth_basis_for_symmetric_tensors (const unsigned int k);
 
+    /*
+    * A class that represents a point in a chosen coordinate system.
+    */
+    template <int dim>
+    class NaturalCoordinate
```

Comment:
Would it make sense to move this class into your new header file as well?

Reply:
Or do you not want to do this because you use `GeometryModel::Interface`?

Reply:
I havn't tested if I could put it there, but besides that I think it is a matter of how you view this header. I personally think it would be good to keep this new header small. This is because I see it as a work around header which might also have been called `enums.h`, which is only needed until we switch to c++11. Do you view it differently, and why would you want to put it there?

Reply:
No, that's fine. I was just wondering whether you had thought about it.